src/nnet2/nnet-component-test.cc

```diff
@@ -859,6 +859,8 @@ int main() {
 
   for (int32 loop = 0; loop < 2; loop++) {
 #if HAVE_CUDA == 1
+    //// Uncomment the following line to expose the bug in UnitTestDropoutComponent
+    //CuDevice::Instantiate().SetDebugStrideMode(true);
```

Comment:
It looks like there is no test code for AddMatMatDivMat.  This is used in the dropout component; it could have a bug.